packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -1306,6 +1306,10 @@ export class ThinEngine extends AbstractEngine {
         this._cachedIndexBuffer = null;
     }
 
+    public createAlignedTypedArray<T extends TypedArray>(type: TypedArrayConstructor<T>, elementCount: number): T {

```

Comment:
I think this deserves a comment as it does not look like there are any alignment here ? 

As we also do not rely on it in the code here, why is it needed on Engine ?

Reply:
Let's discuss about it today.